app/models/series.rb

```diff
@@ -150,4 +151,8 @@ def set_app_version
     return unless new_record?
     self.app_version = PRX::APP_VERSION
   end
+
+  def update_account_for_stories
+    stories.each { |s| s.update_attributes!(account_id: account_id) } if account_id_changed?
```

Comment:
I am a fan of using the bang operation for update_attributes, especially in this kind of case where it will kick out of the each loop, and rollback the save 👍